src/components/PanelGrid/PanelGridItem.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+
+interface PanelGridItemProps {
+  /**
+   * Should only be passed `Panel` components.
+   */
+  children: React.ReactNode;
+  /**
+   * Indicates that the panel is a sidebar.
+   * Currently doesn't actually do anything,
+   * but it's a good semantic hook.
+   */
+  sidebar?: boolean;
```

Comment:
No current plans, it's just a hunch that we'll want to know which panel is acting as the sidebar through more than just positioning.

tbh right now it's a bit dual-minded: we essentially have to specify which panel is the sidebar on the _parent_, because we do so in the grid definition. While a grid child element can indicate which grid template area it wants to be in, it can't define those areas. So choosing e.g. whether the sidebar is on the right or left has to happen at the parent level. So in that sense, we already know (via the parent) which child is the sidebar, and thus could do any necessary styling that way.

It just "feels" weird for a panel itself to have no internal concept of being a sidebar, which is why I left this hook, but… it doesn't actually add anything we can't get from the parent ("sidebar left + first-child" or "sidebar right + last child", for instance).

Want me to remove?

Reply:
Which is to say: parent forces us into position-based determination of which panel is the sidebar anyway, so a semantic specification like this doesn't serve a practical purpose.

Reply:
Yeah, those motivations make sense, but if it doesn't do anything and we don't have concrete plans for when we'd make it do something, it's just dead code that we should omit. We can add it back in if and when we find we'd benefit from it! So yes, please remove - thank you!